packages/app-sdk/src/utils/asset.ts

```diff
@@ -81,10 +81,18 @@ export function getLocalizedAssetImage(
   asset: Asset,
   imageOrientation: ImageOrientation,
   imageType: string,
+  imageTypeFallback: string,
```

Comment:
This should probably be optional, since it is optional everywhere else. That should probably fix the broken tests as well.